diff --git a/index.toml b/index.toml
index 2c684af..3655c61 100644
--- a/index.toml
+++ b/index.toml
@@ -963,7 +963,7 @@ metafile = true
 
 [[files]]
 file = "mods/what-are-they-up-to.pw.toml"
-hash = "bf169dd7307af346fd880833a4adc23a4dfb8051e9703c8519f3a105f4a466b9"
+hash = "bc46ba6776a054d982ac529a25b1364201611ea4c23f336af2479cc51e363004"
 metafile = true
 
 [[files]]
diff --git a/mods/what-are-they-up-to.pw.toml b/mods/what-are-they-up-to.pw.toml
index 628264a..e3fbea2 100644
--- a/mods/what-are-they-up-to.pw.toml
+++ b/mods/what-are-they-up-to.pw.toml
@@ -1,13 +1,7 @@
-name = "What Are They Up To (Watut)"
+name = "what-are-they-up-to"
 filename = "watut-forge-1.20.1-1.1.3.jar"
-side = "both"
 
 [download]
-url = "https://cdn.modrinth.com/data/AtB5mHky/versions/Sp0NpaDe/watut-forge-1.20.1-1.1.3.jar"
-hash-format = "sha1"
-hash = "869600c0d3fc7ca12f534bcc1d89ba03fac3c0c3"
-
-[update]
-[update.modrinth]
-mod-id = "AtB5mHky"
-version = "Sp0NpaDe"
+url = "https://github.com/alikindsys/terratech-ceu/releases/download/fixes-watut/watut-forge-1.20.1-1.1.3.jar"
+hash-format = "sha256"
+hash = "7f47cb364cb72c82a368eb9f2aab8f9c1327e3b033bf305096bdd4118f3da36c"
diff --git a/pack.toml b/pack.toml
index fdf24e9..ff5bfe8 100644
--- a/pack.toml
+++ b/pack.toml
@@ -6,7 +6,7 @@ pack-format = "packwiz:1.1.0"
 [index]
 file = "index.toml"
 hash-format = "sha256"
-hash = "ac12184a46813b8b816226c3ed16d6dc2e3dbd7aac2662d68c7a0ee942c6f5ce"
+hash = "3c9ddca0889f4aa735735af18f6093020b805b5d4b96ae815be7110ea1b536fc"
 
 [versions]
 minecraft = "1.20.1"
